Gemfile

```diff
@@ -2,3 +2,5 @@
 source 'http://rubygems.org'
 
 gemspec
+
+gem 'pry'
```

Comment:
Do we still need this or did you add it for testing?

Reply:
only added for testing

Reply:
nit: add `pry-byebug` instead, it has a bunch of nice features.

Reply:
I'm confused, do we want to leave this in? Is it used in specs?

Reply:
It's not actively used in specs, but I think it would be neat to keep it in there, so that you can just drop `bundle.pry` whenever you need to without modifying your Gemfile and installing new gems